doc/CHANGELOG.md

```diff
@@ -29,7 +35,7 @@ startup. Please note that these directories will not necessarily be part of your
 
 * 19-01-2020
   - The name has been changed to **Zinit** based on the results of the
-    [poll](https://github.com/zdharma-continuum/zinit/issues/235).
+    [poll](https://github.com/zdharma/zinit/issues/235).
```

Comment:
Why don't we just us the target `#history-lost-for-issue-235` or something like that... or (in this case) remove the link.

Reply:
I like the idea.

Rather than just removing it entirely we can create a wiki entry named `null` that explains why this link is dead. And lists a a few hint about where to *maybe* find the missing info regardless:

- google cache
- wayback machine
- etc

Reply:
https://github.com/zdharma-continuum/zinit/issues/50